sig-ui/annual-report-2021.md

```diff
@@ -4,15 +4,14 @@
 
 1. What work did the SIG do this year that should be highlighted?
 
-   -
-   -
-   -
+   - Kubernetes Dashboard v2.2.0-v2.4.0 released in 2021, and latest release is v2.5.0 in Feb 2022. Each releases support latest release of k8s at the time.
+   - Almost dependencies are updated to latest.
+   - In addition to the ongoing effort of maintenance, we are improving support visualizing [various resource objects](https://github.com/kubernetes/dashboard/issues/5232) and adding the ability for installations to have custom themes.
```

Comment:
s/we are improving support visualizing various resource objects/we are improving support for various resource objects